web/src/components/Korrel8rPanel.tsx

```diff
@@ -47,63 +50,74 @@ export default function Korrel8rPanel({ initialQueryString }: Korrel8rPanelProps
     dispatch(setQuery(queryString));
   }
 
+  const preventQuery = React.useMemo(() => {
+    return !query || netobserveAvailableLoading || loggingAvailableLoading;
+  }, [query, netobserveAvailableLoading, loggingAvailableLoading]);
+
+  const handleError = React.useCallback(
+    (error) => {
+      // eslint-disable-next-line no-console
+      console.error(error);
+      setLoadingFalse();
+      let displayError: string;
+      try {
+        displayError = JSON.parse(error.message).error;
+        setErrorMessageTitle(t('Korrel8r Error'));
+      } catch (e) {
+        displayError = error.message;
+        setErrorMessageTitle(t('Error Loading Data'));
+      }
+      setErrorMessage(displayError);
+    },
+    [setLoadingFalse, setErrorMessageTitle, setErrorMessage, t],
+  );
+
+  const clearError = React.useCallback(() => {
+    setErrorMessage('');
+    setErrorMessageTitle('');
+    setLoadingFalse();
+  }, [setErrorMessage, setErrorMessageTitle, setLoadingFalse]);
```

Comment:
these functions do not change so this can just be:

```suggestion
  const clearError = () => {
    setErrorMessage('');
    setErrorMessageTitle('');
    setLoadingFalse();
  }
```